test/integration/apiserver/apiserver_test.go

```diff
@@ -3199,7 +3202,7 @@ func TestEmulatedStorageVersion(t *testing.T) {
 	for emulatedVersion, cases := range groupedCases {
 		t.Run(emulatedVersion, func(t *testing.T) {
 			server := kubeapiservertesting.StartTestServerOrDie(
-				t, nil, []string{"--emulated-version=kube=" + emulatedVersion, `--storage-media-type=application/json`}, framework.SharedEtcd())
+				t, nil, []string{"--emulated-version=kube=" + emulatedVersion, `--storage-media-type=application/json`, fmt.Sprintf("--runtime-config=%s=true", admissionregistrationv1beta1.SchemeGroupVersion)}, framework.SharedEtcd())
```

Comment:
I'm using this pattern with `admissionregistrationv1beta1.SchemeGroupVersion` because then the reference to the API is checked by the compiler at compile time. Removing the API will prevent compilation, whereas a left-over string might only fail at runtime (`admissionregistrationv1beta1` was not used by itself).